pkg/infrastructure/infinispan.go

```diff
@@ -28,15 +28,18 @@ import (
 )
 
 const (
-	// InfinispanOperatorName is the Infinispan Operator default name
-	InfinispanOperatorName = "infinispan-operator"
-	infinispanServerGroup  = "infinispan.org"
-	defaultInfinispanPort  = 11222
+	// infinispanOperatorName is the Infinispan Operator default name
+	infinispanOperatorName = "infinispan-operator"
+	// Default Infinispan server group
+	infinispanServerGroup = "infinispan.org"
```

Comment:
Don't we already have this constant in this package? :thinking: 
Either way we can use `infinspan.GroupVersion.Group.String()`.

Reply:
Changed